deps/llvm.mk

```diff
@@ -73,17 +73,29 @@ LLVM_LIBCXX_TAR:=$(SRCDIR)/srccache/libcxx-$(LLVM_TAR_EXT)
 endif
 endif # LLVM_VER != svn
 
+# Figure out which targets to build (hardcode this to "host;NVPTX" when on 3.9+ only)
+LLVM_TARGETS := host
+ifeq ($(LLVM_VER_SHORT),$(filter $(LLVM_VER_SHORT),3.9 svn))
+LLVM_TARGETS += NVPTX
+endif
+EMPTY :=
+SPACE := $(EMPTY) $(EMPTY)
+ifeq ($(LLVM_USE_CMAKE),1)
+LLVM_TARGET_STRING := $(subst $(SPACE),;,$(LLVM_TARGETS))
+else
+COMMA := ,
+LLVM_TARGET_STRING := $(subst $(SPACE),$(COMMA),$(LLVM_TARGETS))
```

Comment:
do we need to worry about this case? it's not possible to use configure with 3.9 or newer, so I'd simplify this as something like
```
ifeq ($(LLVM_VER_SHORT),$(filter $(LLVM_VER_SHORT),3.3 3.4 3.5 3.6 3.7 3.8))
LLVM_TARGET_STRING := host
else
LLVM_TARGET_STRING := host;NVPTX
endif
```
if that works